install_hortanworks.md

```diff
@@ -61,14 +61,13 @@
 4. Create database
   1. Hive
 ```mysql
-CREATE USER 'hive'@'master' IDENTIFIED BY 'hive';
-GRANT ALL PRIVILEGES ON *.* TO 'hive'@'localhost' with grant option;
+CREATE USER 'hive'@'master2.cloudwick.com' IDENTIFIED BY 'hive';
```

Comment:
Should be done with a variable declaration on line 62 and then concatenation. Hard-coding the value is not a good option. An even better solution would be to take this value from argv. Take a look at the implementation of reverse tunneling. Last but most important, we need to have scripts rather than text. this cannot be run on a cluster. 
This is my opinion, if you disagree I am happy to discuss.